src/gui/widgets/tree_view.cpp

```diff
@@ -59,7 +59,7 @@ tree_view_node& tree_view::add_node(
 	return get_root_node().add_child(id, data, index);
 }
 
-std::pair<tree_view_node::ptr_t, int> tree_view::remove_node(tree_view_node* node)
+std::pair<std::shared_ptr<tree_view_node>, int> tree_view::remove_node(tree_view_node* node)
```

Comment:
Why did the type change here? It looks like you just removed a typedef… why?

Reply:
Because it was barely used and `ptr_t` is so undescriptive that it just ends up taking more time to remember/check what that type actually is than it will ever save anyone in the slightly reduced typing.